tools/pattern-lab/config/config.yml

```diff
@@ -18,7 +19,7 @@ patternStates:
 styleGuideExcludes: ''
 cacheBusterOn: 'true'
 exportDir: export
-publicDir: ../../dist/public/
+publicDir: ../../dist/pl/
```

Comment:
This is a change we'll probably want to make sure is included in the docs.

Reply:
Indeed, but holding off as we experiment and stabilize.